web/.gitignore

```diff
@@ -0,0 +1,24 @@
+# dependencies
+/node
+/yarn
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
```

Comment:
this pattern is already covered by root gitignore

Reply:
Yeah, it was missing when I added it. Fixed